index.d.ts

```diff
@@ -0,0 +1,7 @@
+import { BackendModule } from "i18next";
+
+export function mainBindings(ipcMain: Electron.IpcMain, browserWindow: Electron.BrowserWindow, fs: any): any;
+export function clearMainBindings(ipcMain: Electron.IpcMain): any;
+export function preloadBindings(ipcRenderer: Electron.IpcRenderer, process: NodeJS.Process): any;
+
+export default {} as BackendModule;
```

Comment:
I'm trying to import this module, but only works these ways:
- Import everything (`import * as i18nextBackend from 'i18next-electron-fs-backend`)
- Destructuring (`import { mainBindings, clear...} from 'i18next-electron-fs-backend'`)

Example:
![image](https://user-images.githubusercontent.com/45635827/190921787-4995becc-6950-41f5-b5c8-912018ed4f08.png)

I was viewing the code to check if I could use a default export, and then I found something, is this correct?
![image](https://user-images.githubusercontent.com/45635827/190921727-8d2bd449-0ee2-40e0-b82a-5dedb46d6e71.png)

I'm using TypeScript and when I run `tsc -w` in the terminal this appears
![image](https://user-images.githubusercontent.com/45635827/190921679-dc977632-d8b0-41d2-8263-14841c6682d2.png)

Everything work as expected, but I don't like to see this every time I compile the TS code to JS, and also I would like to use the default export

I was searching on the internet for a solution and I found this:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#handling-many-consuming-import

So you need to change this class:
https://github.com/reZach/i18next-electron-fs-backend/blob/a099cbbea5536b9420b6e4d7c8b9b39827a1b1ba/src/index.js#L103-L387

I'm not expert using exports, but I think this might fix the problem:
```js
class BackendInstance { /* Use this name instead of Backend */
    /*The Backend class code...*/
}
function Backend() {
    return new BackendInstance();
}
Backend.BackendInstance = BackendInstance;
Backend.Backend = Backend; /* Allows for { Backend } */
Backend.default = Backend; /* Allows for strict ES Module support */
module.exports = Backend; /* Sets the default export */
```

If you modify this, I'll appreciate it! 💖

Reply:
Thank you for your message @IvanGodinez21, are you able to create a pull request for these changes?

Reply:
Done check #16 @reZach